opennms-doc/guide-admin/src/asciidoc/text/telemetryd/protocols/nxos/nxos-adapter.adoc

```diff
@@ -0,0 +1,37 @@
+
+===== Cisco NXOS Adapter
```

Comment:
Aside from the title, all of the content here continues to refer to the JTI adapter.

Reply:
I did change the contents. Looks like it didn't end up here.